docs/dictionary/keyword/URL.lcdoc

```diff
@@ -43,27 +44,45 @@ on textRead
-
-From LiveCode 7.0.0, URL keyword has been upgraded to understand Unicode files when using URL("file:"). If the file located at the path has a Byte Order Mark at its beginning, then URL will decode the file according to that Byte Order Mark (UTF-8, UTF-16BE, UTF-16LE, UTF-32BE and UTF-32LE are compatible). If no Byte Order Mark is found, then the file will be decoded as if it were using Native encoding (MacRoman on Mac OS X, ISO-8859-1 on Linux, CP-1252 on Windows).
-
-References: HTMLText (property), URL (keyword), file (keyword), libURLSetAuthCallback (command), launch url (command), libURLSetExpect100 (command), load (command), container (glossary), keyword (glossary), resource (glossary)
+Use the <URL(keyword)> <keyword> to access the contents of a <file|local file> or a <file> accessible on the Web.
+
+A <URL(keyword)> is a method of designating a <file> or other <resource>. You can use a <URL(keyword)> like any other <container>.
+You can get the contents of a <URL(keyword)> or use its contents in any <expression>. LiveCode supports the following
+<URL scheme|URL schemes>: 
+
+* <http>: a page from a <web server>
+* <ftp>: a directory or <file> on an <FTP> server
+* <file>: a <text file> on the local disk (not on a server)
+* <binfile>: a <binary file>
+* <resfile>: on Mac OS and OS X systems, the <resource fork> of a <file>
+
+All actions that refer to a <URL(keyword)> container are blocking: that is, the handler pauses until LiveCode is finished accessing 
+the <URL(glossary)>. Since fetching a web page may take some time due to network lag, accessing URLs may take long enough to be 
+noticeable to the user. To avoid this delay, use the <load> <command> (which is non-blocking) to cache web pages before you need them.
+
+For technical information about <URL(glossary)|URLs> and <URL scheme|URL schemes>, see [RFC 1630](https://tools.ietf.org/html/rfc1630).
+
+>*Important:*  The <http> and <ftp> keywords are part of the <Internet library> on desktop <platform|platforms>. 
+To ensure that these <URL(glossary)> types work in a desktop <standalone application>, you must include this custom <library> when you 
+create your standalone. In the Inclusions section of the Standalone Application Settings window, make sure Internet Library is 
+selected in the list of script libraries. The iOS and Android engines do not support the libUrl <library> but allow you to use 
+<URL(keyword)>. When specifying <URL(glossary)|URLs> for iOS and Android, you must use the appropriate form that confirms to 
+[RFC 1630](https://tools.ietf.org/html/rfc1630). 
+Ensure that you <urlEncode> any username and password fields appropriately for <FTP> urls.
+
+>*Important:*  The space character is not valid in <url(glossary)|URLs>, however the libURL <library> (Desktop platforms) replaces 
```

Comment:
I guess this `<url(glossary)` needs to be `<URL(glossary)` due to case sensitivity in the javascript